src/python/pants/core/util_rules/partitions.py

```diff
@@ -204,3 +228,78 @@ async def partitioner(
         )
 
     return collect_rules(locals())
+
+
+@memoized
+def _partition_per_input_field_sets_partitioner_rules(cls) -> Iterable:
+    """Returns a rule that implements a "partitioner" for `PartitionFieldSetsRequest`, which returns
+    a single-element partition per input."""
```

Comment:
```suggestion
    """
    Returns a rule that implements a "partitioner" for `PartitionFieldSetsRequest`, which returns
    a single-element partition per input.
    """
```

is alot more diff-friendly

Reply:
`docformatter` doesn't allow this one 😞